src/shared/components/Header/WorkspaceSelectorMenu/WorkspaceSelectorMenuButton.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { Button, Popover } from 'antd';
+import { WorkspaceSelectorContainerProps } from './workspaceSelectorContainer';
+import { Organization, Project } from '@bbp/nexus-sdk';
+import WorkspaceSelectorMenu from './WorkspaceSelectMenu';
+
+const getButtonLabel = (
+  activeOrg: Organization | null,
```

Comment:
Any reason why it's not `activeOrg?: Organization`

Reply:
I just like null better. I know I put it there